console/src/main/scala/io/joern/console/cpgcreation/ImportCode.scala

```diff
@@ -21,7 +26,7 @@ class ImportCode[T <: Project](console: io.joern.console.Console[T])(implicit
   protected val generatorFactory = new CpgGeneratorFactory(config)
 
   private def checkInputPath(inputPath: String): Unit = {
-    if (!File(inputPath).exists) {
+    if (!Files.exists(new File(inputPath).toPath)) {
```

Comment:
Go directly to path
```suggestion
    if (!Files.exists(Paths.get(inputPath)) {
```
same elsewhere